themes/digital.gov/layouts/events/card-event.html

```diff
@@ -51,13 +51,6 @@ <h2 class="card-event__title" id="{{ .Title | urlize }}">
         </div>
       {{- end -}}
 
-      {{- if .Params.host -}}
```

Comment:
after reviewing the file this code was getting a display none and looked to not being used at all.